addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -440,6 +440,9 @@ function classToStyle($editable, cssRules) {
         style = style.replace(/!important/g,'');
         for (const [key, value] of Object.entries(css)) {
             if (!(new RegExp(`(^|;)\\s*${key}[ :]`).test(style))) {
+                if (node.matches("table, thead, tbody, tfoot, tr, td, th") && key.includes("color")) { 
+                    continue;
+                }
```

Comment:
Wait but this is going to remove all colors, right? Don't we only want to remove the dark mode ones?

Reply:
The problem here is that every assigned color in one mode will look weird and different than the other modes, you can see that the table written in light mode doesn't render as expected in dark mode

Reply:
Is there a way we could isolate the styles from both the dark mode and light mode and blacklist those _only_, leaving any color set by the user themselves?

Reply:
I updated the fix to blacklist those specific css styles from light/darkmode , personally I don't like the current solution I provided since it needs hardcoded selectors from the css files which means changing any of those selectors or adding new ones will not be detected, what do you think ?

Reply:
I meant isolating them based on the path to the file, for example something like `static/scss/dark.scss`, but I don't think the same exists with `light.scss`. Maybe we could do a compromise on your previous solution which was to remove everything that was applying color through a class. Could you have a look at what kind of classes can be applied using the colorpicker in the full width chatter and in mass_mailing? If we can isolate those based on a naming scheme, hopefully something like `o-color-...`, then we could make a special case for these so that we do not remove them but we still remove other classes that apply color like you used to do. What do you think?

Reply:
I searched for the classes here, and it seems that we can't color anything from table, tbody, thead, tr, td except the td with background-color which is applied inline directly , so I guess in 16.0 at least , we can skip any coloring which matches one of the mentioned elements

Reply:
Ok then, let's do that and recheck in future versions.

Reply:
I updated the fix to make use of both solutions which I think it's cleaner than the first proposed solution , let me know what you think